code/__DEFINES/flags.dm

```diff
@@ -27,18 +27,24 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define DF_ISPROCESSING (1<<2)
 
 //! ## FLAGS BITMASK
-#define CONDUCT_1					(1<<5)		//!  conducts electricity (iron etc.)
-#define NODECONSTRUCT_1				(1<<7)		//!  For machines and structures that should not break into parts, eg, holodeck stuff
-#define OVERLAY_QUEUED_1			(1<<8)		//!  atom queued to SSoverlay
-#define ON_BORDER_1					(1<<9)		//!  item has priority to check when entering or leaving
-#define PREVENT_CLICK_UNDER_1		(1<<11)		//! Prevent clicking things below it on the same turf eg. doors/ fulltile windows
-#define HOLOGRAM_1					(1<<12)
-#define TESLA_IGNORE_1				(1<<13) 	//! TESLA_IGNORE grants immunity from being targeted by tesla-style electricity
-#define INITIALIZED_1				(1<<14)  	//! Whether /atom/Initialize(mapload) has already run for the object
-#define ADMIN_SPAWNED_1				(1<<15) 		//! was this spawned by an admin? used for stat tracking stuff.
-#define PREVENT_CONTENTS_EXPLOSION_1 (1<<16)
-#define UNPAINTABLE_1 				(1<<17)
-#define HTML_USE_INITAL_ICON_1		(1<<18) 			//! Should we use the initial icon for display? Mostly used by overlay only objects
+#define CONDUCT_1					(1<<0)		//!  conducts electricity (iron etc.)
+#define NODECONSTRUCT_1				(1<<1)		//!  For machines and structures that should not break into parts, eg, holodeck stuff
+#define ON_BORDER_1					(1<<2)		//!  item has priority to check when entering or leaving
+#define PREVENT_CLICK_UNDER_1		(1<<3)		//! Prevent clicking things below it on the same turf eg. doors/ fulltile windows
+#define HOLOGRAM_1					(1<<4)
+#define TESLA_IGNORE_1				(1<<5) 	//! TESLA_IGNORE grants immunity from being targeted by tesla-style electricity
+#define INITIALIZED_1				(1<<6)  	//! Whether /atom/Initialize(mapload) has already run for the object
+#define ADMIN_SPAWNED_1				(1<<7) 		//! was this spawned by an admin? used for stat tracking stuff.
+#define PREVENT_CONTENTS_EXPLOSION_1 (1<<8)
+#define UNPAINTABLE_1 				(1<<9)
+#define HTML_USE_INITAL_ICON_1		(1<<10) 			//! Should we use the initial icon for display? Mostly used by overlay only objects
+
+//turf-only flags
+#define NOJAUNT_1					(1<<11)
```

Comment:
Guh this flags_1 stuff is just getting more bloated.

Instead of making the turf-only flags_1 conform to the numeric numbering, why not split them off into turf_flags like tg did? 

Seems a lot less ugly than having so many bitflags

Reply:
(tell me off if this is too big a change ofc, it would be quite a few lines)

Reply:
It's relocation. It's already how it's been already.